developing_packages/package_layout.rst

```diff
@@ -607,3 +607,51 @@ of the ``pkg`` and ``common`` folders, both in the local developer flow in the c
 when those sources are copied to the Conan cache, to be built there with ``conan create`` or ``conan install --build=pkg``.
 This is one of the design principles of the ``layout()``, the relative location of things must be consistent in the user
 folder and in the cache.
+
+
+Environment variables and configuration
```

Comment:
Maybe, it should mention something about `editable mode`. For me, it's quite general that the title and the content is mostly about variables to be used when in editable mode. WDYT?

Reply:
Yes, to be honest, I struggled a bit to find a place for this documentation. I considered the other section more focused on ``editables``, but there very little is explained about the ``layout()`` and it is more operational.

This section already mentions the ``editable`` at least twice, so I thought it was explicit, it is not enough? Any recommendation?

Reply:
Reviewed